src/Psalm/Internal/Stubs/CoreGenericFunctions.php

```diff
@@ -154,3 +154,9 @@ function array_change_key_case(array $arr, int $case = CASE_LOWER) {}
  * @return array<int, array<array-key, T>>
  */
 function array_chunk(array $arr, int $size, bool $preserve_keys = false) {}
+
+
+/**
+* @param resource|HashContext
+*/
+function hash_update($hash, string $data) : bool {}
```

Comment:
I think this should go to `src/Psalm/Internal/CallMap.php` instead. `CoreGenericFunctions` implies generics, i.e. functions with `@template` tags.

Reply:
I can get that done properly later tonight- just did it direct on github because it cropped up during the day job.

Does the CallMap allow per-php-version typing btw?

Reply:
It's possible upstream (in phan/PHPSignatures), but not in Psalm as far as I'm aware. Would require at least #1191.